Update file1.adoc (#17)
alxlenc/pygh-tests: content/file1.adoc

## This is a Title
I create a file.

> A link to the list of drivers and their snapshot support is provided in the references section.

Reviewer: How about...
```suggestion
Here, find the list of drivers with snapshot support in the references section (refer to the link provided).
```
